server/controllers/cohort.js

```diff
@@ -242,7 +245,7 @@ const deleteCohortById = (req, resp) => {
         return resp.sendStatus(400);
       }
 
-      resp.send();
+      resp.send(cohortMembers);
```

Comment:
Please see my comment about the cohort details response.

Reply:
https://github.com/sandstreamdev/eoc/pull/252#discussion_r310105688